### ISSUE TITLE
Can't I just send a message at once?

### ISSUE THREAD

**Issue by kimpig** [jellyfin/jellyfin-plugin-webhook#124]
Emby has the ability to send added media all at once.

The current webhook sends the added media one by one, is there any way to send it similarly?

![이미지 2022-06-18_00-07-39](https://user-images.githubusercontent.com/61234733/177006116-178244f1-32ee-46de-b681-a0c3f5e60342.png)

**Comment by Fallenbagel** [comment]
This might not be a permanent solution (I am working on adding such a feature to this plugin) but for the time being you could try my man in the middle script for this plugin:
[webhook-consolidator](https://github.com/Fallenbagel/webhook-consolidator)
which would send just one message after accumulating and merging all the notifications sent by the plugin within a set period of time (schedule).
so it looks like this:
![image](https://user-images.githubusercontent.com/98979876/217665928-3313643e-746c-479f-b1a1-0dfdb76af623.png)

Currently only telegram & ntfy & discord is supported but I'm adding other notification agents

**Comment by KimPig** [comment]
> This might not be a permanent solution (I am working on adding such a feature to this plugin) but for the time being you could try my man in the middle script for this plugin: [webhook-consolidator](https://github.com/Fallenbagel/webhook-consolidator) which would send just one message after accumulating and merging all the notifications sent by the plugin within a set period of time (schedule). so it looks like this: ![image](https://user-images.githubusercontent.com/98979876/217665928-3313643e-746c-479f-b1a1-0dfdb76af623.png)
> 
> Currently only telegram & ntfy & discord is supported but I'm adding other notification agents

Thank you for coming up with a new solution.

As I read it, it seems that the python script runs independently, so can it be made in Docker form?

**Comment by Fallenbagel** [comment]
> As I read it, it seems that the python script runs independently, so can it be made in Docker form?

Yes both the `main.py` and the notification agent of your choice had to be run manually. You can run it by appending `&` to run it detached, however as I say this I realise docker would be easier 🤔

Sure! I can push a dockerfile and whip up some images tonight c:
